feg/gateway/sbi/sbi_test.go

```diff
@@ -18,93 +18,40 @@ import (
 	"testing"
 	"time"
 
-	"github.com/labstack/echo/v4"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"magma/feg/gateway/sbi"
-	n7_client "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControl"
-	n7 "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControlServer"
-	sbi_common "magma/feg/gateway/sbi/specs/TS29571CommonData"
+	"magma/feg/gateway/sbi/mocks"
+	sbi_NpcfSMPolicyControl "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControl"
+	sbi_CommonData "magma/feg/gateway/sbi/specs/TS29571CommonData"
```

Comment:
In some places `sbi_CommonData` is used and in some places `sbiCommonData`. Better to have a uniform convention.

Reply:
you are right! let me fix that.

I think to avoid confusion and highlight the name lets use the _ one.
This we will know that when we have an `sbi_` that means this is  sbi autogenerated code

Reply:
in fact we are just replacing TSnumber for sbi